### PR TITLE
add cdc workload

### PR DESCRIPTION
**Pull request by cwen0**
Signed-off-by: cwen0 <cwenyin0@gmail.com>

### What problem does this PR solve? 

add cdc workload 

### Check List 

Tests 

 - Unit test
 - E2E test
 - Manual test (add detailed scripts or steps below)
 - No code

Code changes

 - Has Go code change
 - Has CI related scripts change
 - Has Terraform scripts change

Side effects

 - Breaking backward compatibility

Related changes

 - Need to update the documentation

### Does this PR introduce a user-facing change?:

```release-note
NONE
```

### CODE REVIEW

File: argo/template/cdc.yaml [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+          - name: image_version
+            default: nightly
+          - name: storage_class
+            default: standard
+          - name: nemesis
+            default: ""
+          - name: run_time
+            default: "48h"
+          - name: cdc_hub
+            default: "hub.pingcap.net"
```
Comment (mahjonp) [inline argo/template/cdc.yaml]: use docker.io?

Comment (yeya24) [inline argo/template/cdc.yaml]: This should be updated as well.

File: argo/template/cdc.yaml [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+            -run-time={{inputs.parameters.run_time}} \
+            -cdc.hub={{inputs.parameters.cdc_hub}} \
+            -cdc.repository={{inputs.parameters.cdc_repository}} \
+            -cdc.version={{inputs.parameters.cdc_version}} \
+            -loki-addr={{inputs.parameters.loki-addr}} \
+            -loki-username={{inputs.parameters.loki-username}} \
+            -loki-password={{inputs.parameters.loki-password}}
+      retryStrategy:
+        limit: 0
+        retryPolicy: Always
```
Comment (yeya24) [inline argo/template/cdc.yaml]: It is recommended to remove the retryStrategy.

File: argo/workflow/cdc.yaml [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+    - name: call-export-logs
+      steps:
+        - - name: call-export-logs
+            templateRef:
+              name: export-logs
+              template: export-logs
+            arguments:
+              parameters:
+                - name: ns
+                  value: tipocket-cdc
```
Comment (yeya24) [inline argo/workflow/cdc.yaml]: You could use `"{{workflow.parameters.ns}}"` as well.

File: argo/template/cdc.yaml (hunk 1 above)
Comment (cwen0) [reply argo/template/cdc.yaml]: OK

File: argo/workflow/cdc.yaml [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+  generateName: tipocket-cdc-
+  namespace: argo
+spec:
+  entrypoint: call-tipocket-cdc
+  onExit: call-export-logs
+  arguments:
+    parameters:
+        - name: ns
+          value: tipocket-cdc
+            - name: purge
```
Comment (mahjonp) [inline argo/workflow/cdc.yaml]: align parameters?

**Comment by cwen0** [comment]
@mahjonp @yeya24 PTAL again, thanks!

File: argo/workflow/cdc.yaml [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+      - name: nemesis
+        value: ""
+      - name: run_time
+        value: "60m"
+      - name: cdc_hub
+        default: "hub.pingcap.net"
+      - name: cdc_repository
+        default: "qa"
+      - name: cdc_version
+        default: "master"
```
Comment (yeya24) [inline argo/workflow/cdc.yaml]: Also update these? 😄

Comment (cwen0) [reply argo/workflow/cdc.yaml]: 😅Done

File: argo/workflow/cdc.yaml [hunk 6]
```diff
@@ -19,11 +19,11 @@ spec:
       - name: run_time
         value: "60m"
       - name: cdc_hub
-        default: "hub.pingcap.net"
+        value: "hub.pingcap.net"
```
Comment (mahjonp) [inline argo/workflow/cdc.yaml]: also need update these value here.

Comment (mahjonp) [reply argo/workflow/cdc.yaml]: docker.io and pingcap etc.

**Pull request merged**